rflx/specification/style.py

```diff
@@ -62,18 +91,37 @@ def _append(error: RecordFluxError, message: str, row: int, col: int, spec_file:
     )
 
 
+def _determine_enabled_checks(error: RecordFluxError, line: str, spec_file: Path) -> set[Check]:
+    checks = {c.value for c in Check.__members__.values()}
+    disabled_checks = set()
+
+    if "-- style:" in line:
+        m = re.match(r"^-- style: disable = ([^.]*)$", line)
```

Comment:
We could be a bit more forgiving regarding the suppression and allow arbitrary white space before and after `--`, `:`, `=` and at the line ending.

Reply:
Do we want to be that flexible? This would mean to allow 

```
--style:disable=a,b,c

package ...
```
as well as
```
--      style     :disable     =a, b, c

package ...
```

While I personally don't like the former, allowing the latter doesn't make much sense to me. How about the following version: r"^\s*-- {0,2}style: ?disable ?= ?([^.]*)$"?

Reply:
IMHO magic comments are still comments so there should be no restriction of any kind on formatting :thinking:

Reply:
I could live with the version that accepts no white space; at least it's clear what you have to write. The versions that allow only some specific forms of white space are a bit weird to me, and may lead to situations where you stare at your screen and don't understand why it's not working. 
It's also a bit ironic to have strong style requirements on a directive to disable style checks :-)
In other words, I would vote for the simplest form (no spaces), or the most permissive form.

Reply:
As the majority is in favor of allowing arbitrary whitespaces, I have changed the code accordingly.

Reply:
> It's also a bit ironic to have strong style requirements on a directive to disable style checks :-)
> In other words, I would vote for the simplest form (no spaces), or the most permissive form.

I changed my mind - we should torture people trying to disable style checks as much as possible. The comment should start a column 7, followed by exactly 5 spaces and one tab etc. :wink: 

Seriously , relaxing the requirements as much as possible but following @treiher's original solution as a convention will minimize surprises.